src/lib/PageTracker.js

```diff
@@ -0,0 +1,410 @@
+import uuidv4 from 'uuid/v4';
```

Comment:
I think this is a new dependency and is already outdated. Do we need it or can we reuse the `uuidv4` utility in `lib/logUtils`?